lxd/network/network_utils.go

```diff
@@ -1463,6 +1463,11 @@ func ProxyParseAddr(data string) (*deviceConfig.ProxyAddress, error) {
 func AllowedUplinkNetworks(s *state.State, projectConfig map[string]string) ([]string, error) {
 	var uplinkNetworkNames []string
 
+	// There are no allowed networks if project is restricted and restricted.networks.uplinks is not set.
```

Comment:
ideally this would have been in a separate PR so we can merge and test it earlier.